### PR TITLE
feat(user-agent): add User-Agent plugin

### PR DESCRIPTION
**Pull request by helderdiniz**
A plugin that might be interesting for the framework, the cordova-useragent. It's based on https://github.com/LouisT/cordova-useragent

### CODE REVIEW

File: src/@ionic-native/plugins/user-agent/index.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import {Injectable} from '@angular/core';
+import {Plugin, Cordova, IonicNativePlugin} from '@ionic-native/core';
+
+/**
+ * @name User-Agent
```
Comment (ihadeed) [inline src/@ionic-native/plugins/user-agent/index.ts]: This should be `User Agent` without the dash in between. Was this automatically generated by the `gulp plugin:create` command?

File: src/@ionic-native/plugins/user-agent/index.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import {Injectable} from '@angular/core';
+import {Plugin, Cordova, IonicNativePlugin} from '@ionic-native/core';
```
Comment (ihadeed) [inline src/@ionic-native/plugins/user-agent/index.ts]: Please add spaces between the import names and the brackets. A space after `{` and a space before `}`.

**Comment by helderdiniz** [comment]
Why did the build fail if all I did was add whitespaces? It's building correctly on my side.

**Comment by ihadeed** [comment]
@helderdiniz there wasn't an issue with your code, it was just mistake with the latest CI tests script I published. Everything is fine. Thanks for the PR :)

**Comment by helderdiniz** [comment]
Glad to know then! Hope to see this soon, cheers and thanks for the good work on the framework.

**Pull request merged**